### PR TITLE
(BOLT-906) Clarify README

### PR DESCRIPTION
**Pull request by MikaelSmith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,33 +18,45 @@ This library handles parsing JSON input, serializing the result as JSON output,
...
   "input_method": "stdin"
 }
 ```
-and write your task to extend the `TaskHelper` class.
+
+## Usage
+
+When writing your task include the library in your script, extend the `TaskHelper` module, and write the `task()` function. The `task()` function **must** take a hash of parameters as input (even if it's an empty hash), and **must** return a hash. The following is an example of a task that uses the library
+
 ```
```
Comment (donoghuc) [inline README.md]: You can add syntax highlighting for python/json in github markdown. 

```python
import antigravity
```

**Review by donoghuc — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -18,45 +18,50 @@ This library handles parsing JSON input, serializing the result as JSON output,
...
-To use this library in a task, import it in your `Puppetfile`
-```
+To use this library, include this module in a `Puppetfile`
+```ruby
 mod 'puppetlabs-python_task_helper'
 ```
-add it to your metadata
-```
+
+Add it to your [task metadata(https://puppet.com/docs/bolt/1.x/writing_tasks.html#concept-677) (note that the helper expects to read arguments on stdin)
```
Comment (lucywyman) [inline README.md]: Missing a ] after metadata

**Review by lucywyman — APPROVED** [review]
With typo removed lgtm

File: tests/test_mytask.py [hunk 3]
```diff
@@ -7,5 +7,5 @@
 
 class TestHelper(unittest.TestCase):
     def test_task(self):
-        result = MyTask().task({'hello': 'world'})
-        self.assertEqual(result, {'result': 'world'})
+        result = MyTask().task({'name': 'Robert'})
```
Comment (lucywyman) [inline tests/test_mytask.py:10]: Little Bobby Tables we call him :)

**Review by lucywyman — APPROVED** [review]

**Pull request merged**